pyproject.toml

```diff
@@ -119,8 +119,23 @@ fix = true
 include = ["*.py"]
 
 [tool.ruff.lint]
-select = ["W", "E", "F"]
+select = ["D", "W", "E", "F"]
 ignore = [
+    "D100", # Missing docstring in public module"
+    "D101", # Missing docstring in public class"
+    "D102", # Missing docstring in public method
+    "D103", # Missing docstring in public function
+    "D104", # Missing docstring in public package"
+    "D105", # Missing docstring in magic method"
+    "D107", # Missing docstring in `__init__`"
+    "D200", # One-line docstring should fit on one line"
+    "D203", # one-blank-line-before-class and `no-blank-line-before-class` (D211) are incompatible.
+    "D205", # 1 blank line required between summary line and description
+    "D212", # multi-line-summary-first-line. Alternative is to ignore D213 `multi-line-summary-second-line`.
```

Comment:
I'm proposing D213 over D212. Again - I don't care so much about the specific rule as I care about having the rule and consistency.